packages/typespec-python/src/types.ts

```diff
@@ -302,20 +304,36 @@ function emitConstant(type: SdkConstantType) {
     });
 }
 
+function emitEnumValueType(type: SdkEnumValueType): Record<string, any> {
```

Comment:
we are already emitting an enumvalue somewhere so we should combine this code. Also why are we making the type of an `enumvalue` a `constant`?

Reply:
For now, only discriminator use enumvalue so at first I just want to treat it as `constant`. But after offline discussion with @tadelesh, we had better to add that type in the long run. And it is not urgent, so I have more time to complete the code.
Will update later.